WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -374,7 +374,7 @@ private extension StoreStatsV4PeriodViewController {
 
     func configureChart() {
         lineChartView.marker = StoreStatsChartCircleMarker()
-        lineChartView.chartDescription?.enabled = false
+        lineChartView.chartDescription.enabled = false
```

Comment:
A way to double check these changes is to open the diff between versions 3.6.0 and 4.0.3 and do a string search for the changes.

https://github.com/danielgindi/Charts/compare/v3.6.0...v4.0.3

Reply:
Nice way for checking the changes between 2 versions 👍